app/dashboard/_components/InterviewsTable/NetworkSummary.tsx

```diff
@@ -0,0 +1,205 @@
+import type { Codebook, NcNetwork } from '@codaco/shared-consts';
+import { cn } from '~/utils/shadcn';
+
+type NodeColorSequence =
+  | 'node-color-seq-1'
+  | 'node-color-seq-2'
+  | 'node-color-seq-3'
+  | 'node-color-seq-4'
+  | 'node-color-seq-5'
+  | 'node-color-seq-6'
+  | 'node-color-seq-7'
+  | 'node-color-seq-8';
+
+type EdgeColorSequence =
+  | 'edge-color-seq-1'
+  | 'edge-color-seq-2'
+  | 'edge-color-seq-3'
+  | 'edge-color-seq-4'
+  | 'edge-color-seq-5'
+  | 'edge-color-seq-6'
+  | 'edge-color-seq-7'
+  | 'edge-color-seq-8'
+  | 'edge-color-seq-9';
+
+type NodeSummaryProps = {
+  color: NodeColorSequence;
+  count: number;
+  typeName: string;
+};
+
+type EdgeSummaryProps = {
+  color: EdgeColorSequence;
+  count: number;
+  typeName: string;
+};
+
+function NodeSummary({ color, count, typeName }: NodeSummaryProps) {
+  const classes = cn(
+    'flex items-center h-8 w-8 justify-center rounded-full',
```

Comment:
I think we should add the colors from `node-color-seq-1` to the base classes just for the scenario where there are more than 10 node types for some reason. Also for edges.